ChiTech/ChiMesh/MeshContinuum/chi_meshcontinuum_exportvtk.cc

```diff
@@ -131,12 +121,11 @@ void chi_mesh::MeshContinuum::ExportCellsToVTK(const char* baseName)
         d_node[1] = grid->vertices[vgi].y;
         d_node[2] = grid->vertices[vgi].z;
 
-        points->InsertPoint(nc,d_node.data());
-        cell_info[v] = nc; nc++;
+        points->InsertPoint(node_count, d_node.data());
+        cell_info[v] = node_count++;
```

Comment:
++node_count or node_count++? and why??

Reply:
I think this is here because this PR was based on the #152, which was just merged.